### PR TITLE
{perf}[GCC/11.3.0] kineto v0.4.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3161.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/83c490b5141b20295a7710de2a6a9f0b for a full test report.

**Comment by terjekv** [comment]
Test report by @terjekv
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
ninhursaga.uio.no - Linux RHEL 9.1 (Plow), x86_64, Intel(R) Xeon(R) Silver 4216 CPU @ 2.10GHz, 1 x NVIDIA Tesla T4, 525.60.13, Python 3.9.14
See https://gist.github.com/5349bba7168386b95bd8b9c02c703b8a for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04.2 LTS (Focal Fossa), x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/169f212075ab26da27d65fc9e2ffe63a for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/931dd833ac7e8efbab0ad332a09e57dd for a full test report.

File: easybuild/easyconfigs/k/kineto/kineto-0.4.0-GCC-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        'tag': 'v%(version)s',
+        'recursive': True,
+    },
+    'filename': SOURCE_TAR_GZ,
+}]
+checksums = [None]
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+]
```
Comment (branfosj) [inline easybuild/easyconfigs/k/kineto/kineto-0.4.0-GCC-11.3.0.eb:25]:
```suggestion
    ('Python', '3.10.4', '-bare'),
]
```

From the buildlog
```
-- Found PythonInterp: /rds/projects/2017/branfosj-rse/easybuild/venv-dev-Ubuntu20.04/bin/python (found version "3.8.5")
```

And we see that used at https://github.com/pytorch/kineto/blob/main/libkineto/CMakeLists.txt#L14

**Review by branfosj — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/01e1cb456a55bf55028d02270509bbf2 for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**